src/ZoneServer/Buffs/Handlers/Scout/Assassin/Hasisas.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Reflection.Metadata;
+using Melia.Shared.Game.Const;
+using Melia.Zone.Buffs.Base;
+
+namespace Melia.Zone.Buffs.Handlers.Scout.Assassin
+{
+	/// <summary>
+	/// Handle for the Bear Buff, which increases the target's defense.
+	/// </summary>
+	[BuffHandler(BuffId.Hasisas_Buff)]
+	public class Hasisas : BuffHandler
+	{
+		private const string aspdVarName = "Melia.AttackSpeedBonus";
+		private const string critVarName = "Melia.CritDamageBonus";
+		private const float aspdBonusBase = 155;
+		private const float aspdBonusPerLevel = 20;
+		private const float critBonusBase = 3f;
+		private const float critBonusPerLevel = 1.5f;
+
+		public override void OnStart(Buff buff)
+		{
+			var aspdBonus = aspdBonusBase + aspdBonusPerLevel * buff.NumArg1;
+			buff.Vars.SetFloat(aspdVarName, aspdBonus);
+
+			var critBonus = critBonusBase + critBonusPerLevel * buff.NumArg1;
+			buff.Vars.SetFloat(critVarName, critBonus);
+
+			buff.Target.Properties.Modify(PropertyName.ASPD_BM, aspdBonus);
+			buff.Target.Properties.Modify(PropertyName.CRTATK_BM, critBonus);
+		}
+
+		public override void OnEnd(Buff buff)
```

Comment:
Similarly to the skill no longer saying that it consumes a Hasisas Potion, it seems like this also used to
> continuously consume your HP as a side effect

Is this also no longer a thing? Though I did check the buff description, and there the health loss is mentioned.

Reply:
Definitely seems like this was removed, I note in the old buff description, it also increased your crit damage by proportion to your missing health, but that whole part of the description is gone.

Reply:
Just checked the latest description of the buff once more to be absolute certain, and it does still state that it uses HP.
> Increases basic attack speed and critical damage. Reduces HP over time

Unless that effect is not present on officials, we probably want to add it to the buff, right? That will also make it less weird that you no longer need an item to use it.

Reply:
Checked on official and seems it does not consume hp overtime or upon cast at all.
Buff tooltip still says it does, however.